etc/config/c++.amazon.properties

```diff
@@ -1304,6 +1319,19 @@ compiler.arm64g1220.demangler=/opt/compiler-explorer/arm64/gcc-12.2.0/aarch64-un
 compiler.arm64gtrunk.exe=/opt/compiler-explorer/arm64/gcc-trunk/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-g++
 compiler.arm64gtrunk.semver=trunk
 
+###############################
+# GCC Morello
+group.gcc64marm.groupName=Arm Morello GCC
+group.gcc64marm.compilers=arm64gm101a2
+group.gcc64marm.isSemVer=true
+group.gcc64marm.objdumper=/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp2-x86_64/aarch64-none-elf/bin/objdump
+group.gcc64marm.demangler=/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp2-x86_64/aarch64-none-elf/bin/c++filt
```

Comment:
Note: exe not present.

Reply:
Probably should be: `/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp2-x86_64/bin/aarch64-none-elf-c++filt`

Reply:
```suggestion
group.gcc64marm.objdumper=/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp2-x86_64/bin/aarch64-none-elf-objdump
group.gcc64marm.demangler=/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp2-x86_64/bin/aarch64-none-elf-c++filt
```